src/Confluent.Kafka/WatermarkOffsets.cs

```diff
@@ -28,5 +28,8 @@ public WatermarkOffsets(Offset low, Offset high)
 
         public Offset Low { get; }
         public Offset High { get; }
+
+        public override string ToString()
+            => $"[{Low}, {High}]";
```

Comment:
This looks more like a list with elements rather than a range.
Maybe "L-H" or "L..H" ?